app/router.js

```diff
@@ -7,15 +7,10 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
-    this.route('preprints', { path: '/preprints/:file_id' }, function() {
-        this.route('view', { path: '/' });
-        this.route('edit', { path: '/edit' });
```

Comment:
> No edit route?

From the PR description:
"Removes the edit page entirely since it doesn't work."

From the JIRA ticket:
 "`/preprints/preprints/:pre_id/edit` —> Delete it for initial release (page doesn’t work and behavior unclear). Eventually move to `/preprints/content/:pre_id/edit`"